components/logins/ffi/src/lib.rs

```diff
@@ -158,6 +158,7 @@ pub extern "C" fn sync15_passwords_wipe_local(handle: u64, error: &mut ExternErr
 #[no_mangle]
 pub extern "C" fn sync15_passwords_reset(handle: u64, error: &mut ExternError) {
     log::debug!("sync15_passwords_reset");
+    // XXX - does this really need to be exposed?
```

Comment:
Yes, iOS needs it, and it's useful for everybody that doesn't adopt a lockbox-esque model of 'sign out of sync == delete everything locally'. Right now it's deprecated in the android but #649 is to undo that.